tipg/model.py

```diff
@@ -86,7 +86,7 @@ class Collection(BaseModel):
     links: List[Link]
     extent: Optional[Extent]
     itemType: str = "feature"
-    crs: List[str] = ["http://www.opengis.net/def/crs/EPSG/0/4326"]
+    crs: List[str] = ["http://www.opengis.net/def/crs/OGC/1.3/CRS84"]
```

Comment:
the default should be `CRS84`. Now the issue is that PostGIS will have the srid of the collection set to `4326` but all the input/output are in WGS84 (**lon/lat**) 🤷‍♂️